docs/pages/resources/profiles.md

```diff
@@ -322,108 +322,28 @@ evolved from this profile to conform with RO-Crate 1.1's
 
 ## ARC RO-Crate profile
 
-A profile of RO-Crate for [Annotated Research Contexts](https://nfdi4plants.org/content/learn-more/annotated-research-context.html) (ARC), developed by [DataPLANT](https://nfdi4plants.org/).
+The [ARC RO-Crate profile](https://github.com/nfdi4plants/arc-ro-crate-profile) describes [Annotated Research Contexts](https://arc-rdm.org/) (ARC), developed by [DataPLANT](https://nfdi4plants.org/).
 An ARC consists of [ISA](https://isa-specs.readthedocs.io/en/latest/isamodel.html) metadata describing the experimental setup and computational workflows given in CWL.
-The current [profile](https://github.com/nfdi4plants/ARC-specification/blob/main/ARC%20specification.md#appendix-conversion-of-arcs-to-ro-crates) requires the crate to follow the ISA Investigation profile on the top level.
-In the future, the ARC profile will be extended to not only cover the ISA part of an ARC, but also computational workflows, following the existing profiles for this kind of data.
+The current profile is a combination of the [ISA RO-Crate profile](https://github.com/nfdi4plants/isa-ro-crate-profile) and the [ARC CWL RO-Crate profile](https://github.com/nfdi4plants/arc-cwl-ro-crate-profile), see details below.
+It requires the crate to follow the ISA Investigation profile on the top level.
+The investigation object contains studies and assays from the ISA model, as well as computational workflows.
+The core entity of both sub-profiles is the type `LabProcess` defined by [bioschmeas](https://bioschemas.org/) and used to describe processes leading to data files in a parametrized way.
```

Comment:
```suggestion
The core entity of both sub-profiles is the type `LabProcess` defined by [Bioschemas](https://bioschemas.org/) and is used to describe processes leading to data files in a parametrized way.
```

Reply:
To me, the second 'is' seems wrong here. Maybe change it to "The core entity of both sub-profiles is the type `LabProcess` defined by Bioschemas, which is used to describe processes leading to data files in a parametrized way"?

Reply:
sure, updated!